velox/docs/monthly-updates/may-2024.rst

```diff
@@ -0,0 +1,80 @@
+***************
+May 2024 Update
+***************
+
+Documentation
+=============
+
+* Publish blog post about `optimizing TRY and TRY_CAST <https://velox-lib.io/blog/optimize-try-more>`_
+* Publish `Technical Governance <https://velox-lib.io/docs/community/technical-governance>`_.
+
+Core Library
+============
+
+* Optimize TRY and TRY_CAST for cases when many rows fail.
+* Add support for LEFT SEMI FILTER and RIGHT SEMI FILTER merge joins.
+* Add support for DECIMAL input to aggregations over distinct values. :pr:`9850`
+* Add support for specifying overwrite behavior when registering simple scalar functions. :pr:`9158`
+* Fix LEFT merge join with extra filter. :pr:`9862`
+* Fix Nested Loop join with empty build and extra filter. :pr:`9892`
+* Fix crash when copying complex vectors. :pr:`9725`
+* Fix reuse of LazyVectors in TableScan to avoid crashes in downstream operators. :pr:`9811`
```

Comment:
should we add:

Introduce support for adding targeted hierarchical context to runtime exceptions, enabling the inclusion of operator metadata to easily identify the operator throwing the exception #9695